catalyst/contrib/dl/callbacks/trace.py

```diff
@@ -0,0 +1,130 @@
+from typing import Union  # isort:skip
+from operator import lt, gt
+
+
+from pathlib import Path
+
+from catalyst.dl import Callback, CallbackOrder, State
+from catalyst.dl.utils import (
+    trace_model_from_state,
+    save_traced_model,
+)
+
+
+class TracerCallback(Callback):
+
+    def __init__(
+            self,
+            metric_key: str,
+            stage: str = None,
+            mode: str = "max",
+            method_name: str = "forward",
+            requires_grad: bool = False,
+            opt_level: str = None,
+            loader: Union[str, int] = None,
+            trace_mode: str = "eval",
+            out_dir: Union[str, Path] = None,
+            out_model: Union[str, Path] = None,
+    ):
+        """
+        Traces model using created experiment and runner.
+
+        :param metric_key (str): Metric key we should trace model based on
+        :param mode (str): Metric max or min value affects tracing.
+        :param method_name (str): Model's method name that will be
+            used as entrypoint during tracing
+        :param checkpoint_name (str): Checkpoint's name to trace
+        :param stage (str): Stage from experiment from which model and loader
+            will be taken
+        :param loader (str): Loader name to get the batch from
+        :param trace_mode (str): Mode for model to trace
+            (``train`` or ``eval``)
+        :param requires_grad (bool): Flag to use grads
+        :param opt_level (str): AMP FP16 init level
+        :param out_dir (str): Directory to save model to
+        :param out_model: Path to save model to (override out_dir)
+        """
```

Comment:
could you please rewrite doc with Google style format?
Easy-to-go setup you can see here - https://github.com/catalyst-team/catalyst/blob/master/CONTRIBUTING.md#documentation

Reply:
ok, will redo :)

Reply:
Done!